pkg/common/peertracker/tracker_linux.go

```diff
@@ -9,19 +9,30 @@ import (
 	"strings"
 	"sync"
 	"syscall"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spiffe/spire/pkg/common/telemetry"
+)
+
+const (
+	_linuxType = "linux"
```

Comment:
Sorry, i should have pointed this one out as well.

Reply:
I should have been more attentive to seeing other places to follow through :)